src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -1,75 +1,141 @@
-      ['organization', `/organizations/${orgId}/`],
+      ['integrations', `/organizations/${orgId}/integrations/`],
     ];
   }
 
+  mergeIntegration = integration => {
+    // Merge the new integration into the list. If we're updating an
+    // integration overwrite the old integration.
+    const keyedItems = keyBy(this.state.integrations, i => i.id);
+    const integrations = sortArray(
+      Object.values({...keyedItems, [integration.id]: integration}),
+      i => i.name
+    );
+    this.setState({integrations});
+  };
+
+  handleDeleteIntegration = integration => {
+    const {orgId} = this.props.params;
+    addLoadingMessage();
+
+    const options = {
+      method: 'DELETE',
+      success: () => {
+        const integrations = this.state.integrations.filter(i => i.id !== integration.id);
+        this.setState({integrations});
+        addSuccessMessage(t('Integration removed'));
+      },
+      error: () => addErrorMessage(t('Failed to remove Integration')),
+    };
+
+    this.api.request(`/organizations/${orgId}/integrations/${integration.id}/`, options);
+  };
+
+  handleDisableIntegration = integration => {
+    let url;
+    if (integration.accountType === 'User') {
+      url = 'https://github.com/settings/installations';
+    } else {
+      let orgName = integration.domainName.split('/')[1];
+      url = `https://github.com/organizations/${orgName}/settings/installations`;
+    }
+    window.open(url, '_blank');
+  };
+
   renderBody() {
-    let {location} = this.props;
-    let orgFeatures = new Set(this.state.organization.features);
-    let internalIntegrations = new Set(['jira']);
-
-    const linkPrefix = this.props.linkPrefix ? this.props.linkPrefix : location.pathname;
-
-    const integrations = this.state.config.providers
-      .filter(provider => {
-        return (
-          orgFeatures.has('internal-catchall') || !internalIntegrations.has(provider.key)
-        );
-      })
-      .map(provider => (
-        <PanelItem key={provider.key} align="center">
-          <Box>
-            <PluginIcon size={32} pluginId={provider.key} />
-          </Box>
-          <Box px={2} flex={1}>
-            <ProviderName>
-              <Link to={`${linkPrefix}${provider.key}/`} css={{color: theme.gray5}}>
-                {provider.name}
-              </Link>
-            </ProviderName>
-            <AuthorName>{provider.metadata.author}</AuthorName>
-          </Box>
-          <Box>
-            <Button size="small" to={`${linkPrefix}${provider.key}/`}>
-              {t('Manage')}
-            </Button>
-          </Box>
-        </PanelItem>
-      ));
+    const orgId = this.context.organization.slug;
+    const features = this.context.organization.features;
+    const internalIntegrations = ['jira'];
+
+    const enabledProviders = this.state.config.providers.filter(
+      p => features.includes('internal-catchall') || !internalIntegrations.includes(p.key)
```

Comment:
Could we use `<Feature>` here and remove the need to use org from context directly?

```javascript
renderBody() {
    const {orgId} = this.props.params;
    const internalIntegrations = ['jira'];
 	
	return (
		<Feature feature={['internal-catchall']}>
		 {({hasFeature}) => {
    const enabledProviders = this.state.config.providers.filter(
 	      p => hasFeature || !internalIntegrations.includes(p.key)
```

Reply:
307c5da Turns out we didn't need this as the backend was already enforcing 

https://github.com/getsentry/sentry/blob/498f56a93388ab21a10ae383764334078ec0087a/src/sentry/api/endpoints/organization_config_integrations.py#L15-L27